bundle-workflow/src/test_workflow/config/test_manifest.yml

```diff
@@ -31,6 +33,8 @@ components:
       test-configs:
         - with-security
         - without-security
+      additional-cluster-configs:
+        - "plugins.destination.host.deny_list: [10.0.0.0/8, 127.0.0.1]"
```

Comment:
Why is this a string and not a dictionary?

Reply:
Its a list of strings. Additional settings if required in future can be added. It returns as `['plugins.destination.host.deny_list: [10.0.0.0/8, 127.0.0.1]']` which is added in `opensearch.yml`

Reply:
So these are YAML fragments that go into another yaml file. This should look like this:

```yaml
config:
   tests:
     - with-security
     - without-security
   cluster:
     - plugins.detination.host.deny_list: [ 10.0.0.0/8, 127.0.0.1 ]
  ```

Reply:
+1 for adding these settings as dict. It keeps it clear and avoid excessive parsing logic/errors in the code.

Reply:
FYI for it to be dict the test_manifest.yml will look like:
```yaml
  - name: alerting
    integ-test:
      test-configs:
        - with-security
        - without-security
      additional-cluster-configs:
        plugins.destination.host.deny_list : [10.0.0.0/8, 127.0.0.1]
```
Note the removal of `-`